product-catalog/app/productcatalog/models/RichShippingRate.java

```diff
@@ -0,0 +1,15 @@
+package productcatalog.models;
+
+import io.sphere.sdk.models.Base;
+import io.sphere.sdk.shippingmethods.ShippingRate;
+
+public class RichShippingRate extends Base {
```

Comment:
There is no written convention yet, as we need first to agree on the convention and for that I was waiting to have a complete full functionality to have the whole picture. The `FooData` naming I started using for the models that are sent to the templates, but I think this class is not one of them. For classes that are coming from CTP but we are kind of specific for this shop, I started calling them `ShopFoo`, so it would be `ShopShippingRate` (the `Custom` prefix started becoming weird with Customers, Custom Objects and so on :P). But as I said, nothing is established and it will probably change it the near future.